27 move locate me button

jasonrogena opened the pull request:
Closes #27

jasonrogena committed:
JR|ME: Move Locate Me Button To Top Right Edge

Move the locate-me button to the top right edge of the screen. Fixes
issue #27

Signed-off-by: Jason Rogena <jasonrogena@gmail.com>

jasonrogena committed:
JR|ME: Add Tests for Issue #27

Add tests to check for the location of the locate me button in the code
that fixes issue #27

Signed-off-by: Jason Rogena <jasonrogena@gmail.com>

ukanga commented on app/src/androidTest/java/org/redcross/openmapkit/MapActivityTest.java:
```diff
@@ -0,0 +1,78 @@
+
+        //check position when adding a new node
+        /*TODO: test whether locate-me button is above moveNodeModeBtn after sequence for creating
+         TODO: new node is started*/
```
Can you create an issue for this please and remove the commented code.

jasonrogena committed:
JR: Remove TODOs in MapActivityTest

Remove code that was commented out along with associated TODOs in
MapActivityTest. Move TODOs to issue #40

Signed-off-by: Jason Rogena <jasonrogena@gmail.com>
